lib/phobos_db_checkpoint.rb

```diff
@@ -20,7 +20,7 @@ module PhobosDBCheckpoint
 
   class << self
     attr_reader :db_config
-    attr_accessor :db_config_path, :db_dir, :migration_path
+    attr_accessor :db_config_path, :db_dir, :migration_path, :verbose
```

Comment:
I would like to be able to see stuff happening by the flick of a finger, maybe someone else will find it ugly and cluttering the code. WDYT?